src/app/games/games.service.ts

```diff
@@ -35,6 +35,15 @@ export class GamesService {
             });
     }
 
+    getByTeam(teamId: number): Observable<GameEvent[]> {
```

Comment:
I think this makes more sense in the TeamsService. 
A rule of thumb would be: /api/resource/method -> ResourceService.method()